packages/yourloops/test/integration/pages/notification/notification-for-patient.spec.tsx

```diff
@@ -0,0 +1,122 @@
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { act, within, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { renderPage } from '../../utils/render'
+import { mockPatientLogin } from '../../mock/patient-login.mock'
+import { monitoredPatientAsTeamMember } from '../../mock/patient.api.mock'
+import NotificationApi from '../../../../lib/notifications/notification.api'
+import { type Notification } from '../../../../lib/notifications/models/notification.model'
+import { TeamMemberRole } from '../../../../lib/team/models/enums/team-member-role.enum'
+import { NotificationType } from '../../../../lib/notifications/models/enums/notification-type.enum'
+import { PatientNotification, teamOne, teamTwo } from '../../mock/team.api.mock'
+import TeamAPI from '../../../../lib/team/team.api'
+
+describe('Notification page for patient', () => {
+  const invitationTeam: Notification = {
+    id: '26a11710e98e4',
+    type: NotificationType.careTeamPatientInvitation,
+    metricsType: 'join_team',
+    email: 'ylp.ui.test.67-patient12@diabeloop.fr',
+    creatorId: 'dec0816009d32',
+    date: '',
+    target: { id: '63c7b7989cacc878ecce2c40', name: 'sysReq-67-team2' },
+    role: TeamMemberRole.patient,
+    creator: {
+      userid: 'dec0816009d32',
+      profile: {
+        fullName: 'Backloops test SysReq 67 hcp1',
+        patient: {
+          birthday: '',
+          diagnosisDate: ''
+        },
+        email: ''
+      }
+    }
+  }
+
+  beforeAll(() => {
+    mockPatientLogin(monitoredPatientAsTeamMember)
+    jest.spyOn(TeamAPI, 'getTeams').mockResolvedValue([teamOne, teamTwo])
+    jest.spyOn(NotificationApi, 'getReceivedInvitations').mockResolvedValue([invitationTeam])
+    jest.spyOn(TeamAPI, 'getTeamFromCode').mockResolvedValue(PatientNotification)
+    jest.spyOn(NotificationApi, 'acceptInvitation').mockResolvedValue()
+  })
+
+  it('should be able to accept a team invite', async () => {
+    await act(async () => {
+      renderPage('/notifications')
+    })
+
+    const badgeNotification = screen.getByLabelText('Go to notifications list')
+    expect(badgeNotification).toHaveTextContent('1')
+    const badgeTeam = screen.getByLabelText('Open team menu')
+    expect(badgeTeam).toHaveTextContent('2')
+    const acceptButtonForCancelDialog = screen.getByRole('button', { name: 'Accept' })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏I don't understand this variable name. What is it?